extensions/switch-dns/src/config.ts

```diff
@@ -0,0 +1,9 @@
+export const DNS_LIST = [
+  { title: "Alibaba", dns: "223.5.5.5,223.6.6.6", accessory: "alidns.com" },
+  { title: "Google", dns: "8.8.8.8,8.8.4.4", accessory: "Google" },
+  { title: "Cloudflare", dns: "1.1.1.1,1.0.0.1", accessory: "Cloudflare DNS" },
+  { title: "114", dns: "114.114.114.114,114.114.115.115", accessory: "114 Public DNS" },
```

Comment:
I think this set of choices could perhaps be improved.

I'm not familiar with the Chinese market, and my Chinese is very rusty, but Alibaba and 114 are almost impossible to find any details for on the English speaking public internet. Are these services accessible outside of China? Are they safe to use outside of China?

Separately, the list is missing OpenDNS, which has historically been a well known and trustworthy provider, they were the first custom DNS provider I used probably 15 years ago. I'd have expected a list of options to include them.

However... these issues lead me to think about whether this is the best way to configure this extension. Do we want to add DNS servers for the Indian market? Do we want to add DNS servers for the Russian, Iranian, or North Korean markets that might be sanctioned and having difficulty using others? This all gets difficult and political.

Because of this, I'd propose an alternative: Have a setting for users to provide the custom DNS they want to use, and then select from those options. I think this avoids the issue of which DNS servers are worthy of inclusion, and also prevents users from accidentally setting DNS servers that they may not want to use with one mis-click.

What do you think?

Reply:
Thanks for your suggestion. It's my mistake not considering international universality. Originally, custom configuration is my plan. I will submit a new commit with those changes:  
- Provide `Google`, `Cloudflare`, `Default DNS` as default options
- Support user custom configuration

I think that "one-click switching " is useful for most users (installed this extention). Maybe I don't want extra confirmation to slow down my operation. How do you think?

Reply:
Sounds great!